### PR TITLE
[tests-only][full-ci]Refactor webUILogin/adminBlocksUser.feature:20 to make it follow the current behaviour

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
The changed test was failing because it was not compatible with the current behavior of the system, this PR makes it work according to the current behavior and removes closed issue https://github.com/owncloud/web/issues/4795 from the expected to fail.

## Related Issue

- Part of: https://github.com/owncloud/web/issues/7896

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/30475/13/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUILogin-adminBlocksUser_feature-L20.png
  


</p></details>

**Comment by SwikritiT** [comment]
> 

weird the screenshot shows that the user is navigated to the login page but locally for me they are navigated to the disabled page. Need to look further into it.

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/30569/13/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUILogin-adminBlocksUser_feature-L20.png
  


</p></details>

File: tests/acceptance/pageObjects/loginPage.js [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
   },
   elements: {
     body: 'body',
-    authenticateButton: {
+    authenticateButtonOcis: {
       selector: '//div/button/span[@class="MuiButton-label" and text()="Log in"]',
       locateStrategy: 'xpath'
+    },
+    authenticateButtonOC10: {
+      selector: "//button[@class='login-button']//span[text()='Login']",
```
Comment (saw-jan) [inline tests/acceptance/pageObjects/loginPage.js:15]: will it work?
```suggestion
      selector: "//button//span[text()='Login' or text()='Log in']",
```

File: tests/acceptance/pageObjects/loginPage.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const { join } = require('../helpers/path')
+const { client } = require('nightwatch-api')
```
Comment (saw-jan) [inline tests/acceptance/pageObjects/loginPage.js:2]: still needed?

**Review by saw-jan — COMMENTED** [review]
:+1:

**Review by saw-jan — APPROVED** [review]

**Review by grgprarup — COMMENTED** [review]
lgtm :+1:

**Pull request merged**